requirements.txt

```diff
@@ -1,9 +1,31 @@
-click
-PyYAML
-pydantic
-fastapi
-pandas
-gunicorn
-uvicorn
-pymongo
-mongo_adapter
+anyio==3.7.1 ; python_version >= "3.11" and python_version < "4.0"
```

Comment:
Can this file be removed since we have the `pyproject.toml`? I think you can do a pip install with the toml file as well